### DISCUSSION TITLE
It is possible to make the title screen show the wrong song background

### DISCUSSION THREAD

**Discussion by ItsJustSomeDude** [ppy/osu#15787] [Q&A] [answered]
Came across this accidentally.  If you have a song playing on the title screen, then you open singleplayer, switch to another song from the list, then switch back to the previous song from the list, and go back to the title screen it will show the background from the second song you selected.

If the first song that is playing just has an image background, and the second has a storyboard, after switching back to the first you will see a freeze frame of the second's SB.  If the second has a video, after the switch you will see the second song's background image.

https://user-images.githubusercontent.com/46335020/143323190-d36b8740-1074-4e73-8846-a9425254558d.mp4

Not a major issue ofc, just a weird thing I found that I am completely sure is not intentional.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/15804
